### PR TITLE
changing the ablines to use startOfDay function for start/end date in…

### PR DESCRIPTION
**Pull request by mhines-usgs**
… sequence so they're not starting at 7pm

### CODE REVIEW

File: R/dvhydrograph-data.R [hunk 1]
```diff
@@ -106,7 +106,7 @@ getMaxMinIv <- function(data, stat){
 getStatDerived <- function(data, chain_nm, legend_nm, estimated){
   
   points <- data[[chain_nm]][['points']]
-  points$time <- flexibleTimeParse(points[['time']], timezone=data$reportMetadata$timezone)
+  points$time <- flexibleTimeParse(points[['time']], timezone=data$reportMetadata$timezone, shiftTimeToNoon=FALSE)
```
Comment (mhines-usgs) [inline R/dvhydrograph-data.R:109]: don't shift to noon for dvhydro

File: R/dvhydrograph-render.R [hunk 2]
```diff
@@ -53,8 +53,8 @@ createDvhydrographPlot <- function(data) {
     
     # custom gridlines below approval bar
     plot_object <- plot_object %>% 
-      abline(v=seq(from=startDate, to=endDate, by="days"), lty=3, col="gray", where='first') %>%
-      abline(v=seq(from=startDate, to=endDate, by="weeks"), col="darkgray", lwd=1, where='first')
+      abline(v=seq(from=toStartOfDay(startDate), to=toStartOfDay(endDate), by="days"), lty=3, col="gray", where='first') %>%
```
Comment (mhines-usgs) [inline R/dvhydrograph-render.R:56]: these were goofing up the tickmarks vs dates

File: R/utils-json.R [hunk 3]
```diff
@@ -206,7 +206,7 @@ getApprovalIndex <- function(data, points, chain_nm, approval, subsetByMonth=FAL
   return(dates_index)
 }
 
-getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, point_type=NULL, subsetByMonth=FALSE, approvalsAtBottom=TRUE, applyFakeTime=FALSE, extendToWholeDays=FALSE){
+getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, point_type=NULL, subsetByMonth=FALSE, approvalsAtBottom=TRUE, applyFakeTime=FALSE, extendToWholeDays=FALSE, shiftTimeToNoon=TRUE){
```
Comment (mhines-usgs) [inline R/utils-json.R:209]: default shiftTimeToNoon as true

**Comment by mhines-usgs** [comment]
NEW DVHYDRO -- not at noon
image

NEW UVHYDRO - yes to noon
image

**Comment by lindsayplatt** [comment]
@mhines-usgs what is the criteria for switching to noon vs not switching?? Just trying to get it right :)

**Comment by mhines-usgs** [comment]
The rules are as far as I understand @thongsav-usgs describing them was to default all shifts to noon, so that uvhydro Dv uses the noon shift, and then disable for dvhydo so those values display at midnight

**Comment by lindsayplatt** [comment]
ahhhh OK. That makes sense now thanks :)

**Pull request merged**